jobs/shield-daemon/templates/nginx/config/nginx.conf

```diff
@@ -37,46 +37,32 @@ http {
   gzip_disable       "MSIE [1-6]\.(?!.*SV1)";
 
   upstream shield {
-    server localhost:8080;
+    server 127.0.0.1:8080;
```

Comment:
it was changed to 0.0.0.0 in the time before I merged nginx into daemon job; then changed "back" to the valid as I remembered it. I will revert to `localhost`.